### PR TITLE
Add support for ikki68

### PR DESCRIPTION
**Pull request by spbgzh**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Keyboard firmware after change again

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by zvecr** [comment]
<https://github.com/qmk/qmk_firmware/pull/10569#pullrequestreview-504298308>

Again, please stop making new PRs, and just accept the changes to the same branch. Tracking these changes is becoming a real pain.

**Comment by spbgzh** [comment]
> [#10569 (review)](https://github.com/qmk/qmk_firmware/pull/10569#pullrequestreview-504298308)
> 
> Again, please stop making new PRs, and just accept the changes to the same branch. 
> Sorry i didn't see the message, I will not do it again.

**Comment by spbgzh** [comment]
Sorry, this is my first firmware submission.😂

File: keyboards/wuque/ikki68/readme.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# ikki68
+
+![ikki68](https://ikki68.com/wp-content/uploads/2020/06/cover-ikki68-coronation-2.jpg)
+​
+Hope this kit is the same as ikki, every rebirth will be stronger
+More Info at[wuquestudio](https://shop.wuquestudio.com/).
```
Comment (ridingqwerty) [inline keyboards/wuque/ikki68/readme.md:6]:
```suggestion
More Info at [wuquestudio](https://shop.wuquestudio.com/).
```

**Review by ridingqwerty — CHANGES_REQUESTED** [review]
Whitespace touchup in the readme

**Review by drashna — APPROVED** [review]

**Review by ridingqwerty — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**